packages/web-runtime/tests/unit/container/bootstrap.spec.ts

```diff
@@ -1,116 +1,104 @@
-      app: createApp(defineComponent({})),
-      configurationManager: mockDeep<ConfigurationManager>(),
-      runtimeConfiguration: {
-        apps: ['internalFishy', 'internalValid'],
-        external_apps: [{ path: 'externalFishy' }, { path: 'externalValid' }]
-      },
-      store: undefined,
-      router: undefined,
-      gettext: undefined,
-      supportedLanguages: {}
-    })
-
-    expect(buildApplicationMock).toHaveBeenCalledTimes(4)
-    expect(initialize).toHaveBeenCalledTimes(2)
-    expect(errorSpy).toHaveBeenCalledTimes(2)
-    expect(errorSpy.mock.calls[0][0]).toMatchObject(fishyError)
-    expect(errorSpy.mock.calls[1][0]).toMatchObject(fishyError)
-
-    await announceApplicationsReady({
-      app: mock<App>(),
-      store: createStore(defaultStoreMockOptions),
-      applications
-    })
-    expect(ready).toHaveBeenCalledTimes(2)
-  })
-})
-
-describe('announceCustomScripts', () => {
-  afterEach(() => {
-    document.getElementsByTagName('html')[0].innerHTML = ''
-  })
-
-  it('injects basic scripts', () => {
-    announceCustomScripts({
-      runtimeConfiguration: { scripts: [{ src: 'foo.js' }, { src: 'bar.js' }] }
-    })
-    const elements = document.getElementsByTagName('script')
-    expect(elements.length).toBe(2)
-  })
-
-  it('skips the injection if no src option is provided', () => {
-    announceCustomScripts({ runtimeConfiguration: { scripts: [{}, {}, {}, {}, {}] } })
-    const elements = document.getElementsByTagName('script')
-    expect(elements.length).toBeFalsy()
-  })
-
-  it('loads scripts synchronous by default', () => {
-    announceCustomScripts({ runtimeConfiguration: { scripts: [{ src: 'foo.js' }] } })
-    const element = document.querySelector<HTMLScriptElement>('[src="foo.js"]')
-    expect(element.async).toBeFalsy()
-  })
-
-  it('injects scripts async if the corresponding configurations option is set', () => {
-    announceCustomScripts({ runtimeConfiguration: { scripts: [{ src: 'foo.js', async: true }] } })
-    const element = document.querySelector<HTMLScriptElement>('[src="foo.js"]')
-    expect(element.async).toBeTruthy()
-  })
-})
-
-describe('announceCustomStyles', () => {
-  afterEach(() => {
-    document.getElementsByTagName('html')[0].innerHTML = ''
-  })
-
-  it('injects basic styles', () => {
-    const styles = [{ href: 'foo.css' }, { href: 'bar.css' }]
-    announceCustomStyles({ runtimeConfiguration: { styles } })
-
-    styles.forEach(({ href }) => {
-      const element = document.querySelector<HTMLLinkElement>(`[href="${href}"]`)
-      expect(element).toBeTruthy()
-      expect(element.type).toBe('text/css')
-      expect(element.rel).toBe('stylesheet')
-    })
-  })
-
-  it('skips the injection if no href option is provided', () => {
-    announceCustomStyles({ runtimeConfiguration: { styles: [{}, {}] } })
-    const elements = document.getElementsByTagName('link')
-    expect(elements.length).toBeFalsy()
-  })
-})
+it.todo('fix tests')
```

Comment:
dev leftover?

Reply:
Arghs, totally forgot about this one. Kind of... Needs fixing, although I don't know how because of some weird jest-import-foo which is happening. Maybe @dschmidt has an idea what's happening.

Reply:
Resolved